src/components/utils/i18n.js

```diff
@@ -0,0 +1,32 @@
+import Polyglot from 'node-polyglot'
+
+import { locales, mobileLocales } from '../../../locales'
+import { isDesktop } from './'
+
+const defaultLocale = 'en'
+
+const unsupportedLocale = () => {
+  console.warn('Locale not supported')
+  return defaultLocale
+}
+
+const setLocale = (language) => {
+  if (!language || !language.locale) return defaultLocale
+  return locales[language.locale] ? language.locale : unsupportedLocale()
+}
+
+const setTestLocale = (phrases) => {
```

Comment:
Do you plan to fix this before you merge? I don't think we should set stuff on the `window` in production

Reply:
That's the idea, if I can use `fs` module I would be able to get rid of quite a few lines of code in the `/test` files too.